components/director/internal/domain/runtime/model.go

```diff
@@ -0,0 +1,107 @@
+package runtime
+
+import (
+	"fmt"
+	"time"
+)
+
+type RuntimeModel struct {
+	ID          string              `json:"id"`
+	Name        string              `json:"name"`
+	Description *string             `json:"description"`
+	Tenant      string              `json:"tenant"`
+	Labels      map[string][]string `json:"labels"`
+	Annotations map[string]string   `json:"annotations"`
+	Status      *RuntimeModelStatus `json:"status"`
+	// directive for checking auth
+	AgentAuth interface{} `json:"agentAuth"`
+}
+
+type RuntimeModelStatus struct {
+	Condition runtimeModelStatusCondition `json:"condition"`
+	Timestamp time.Time                   `json:"timestamp"`
+}
+
+func NewRuntimeModel(ID string, name string, description *string, tenant string, labels map[string][]string, annotations map[string]string) *RuntimeModel {
```

Comment:
usage of this constructor is more difficult than creating RuntimeModel object itself